Packs/CommonScripts/Scripts/GenericPollingScheduledTask/GenericPollingScheduledTask.js

```diff
@@ -65,6 +67,25 @@ function finish(playbookId, tag, err, entryGUID) {
 }
 
 
+//replace all occurences of textToReplace with replaceWith string
+String.prototype.replaceAll = function(textToReplace, replaceWith) {
```

Comment:
I prefer not to override the builtin replaceAll command.
This command can be used in different places. And it will be hard to track the root cause.

Reply:
[Done](https://github.com/demisto/content/pull/33368/commits/6d3f76ab87d2ea74caa7cd11fdd271a1e1f4a527)